modules/contract/contracts.js

```diff
@@ -75,18 +74,32 @@ class Contracts {
         await this.deleteContract(args[0]);
     }
 
-    async listEntryPoints(contractPath) {
+    async listEntryPoints(contract) {
+        const tezosNode = this.config.provider;  
         const fs = require('fs');
         const conseiljs = require(CONSEIL_JS);
-        let conseilServer = { 'url': CONSEIL_SERVER_URL, 'apiKey': CONSEIL_SERVER_APIKEY, 'network': TESTNET_NAME };
-        let contractCode, contractAddress;
-        let contractObj = Helper.findKeyObj(config.contracts, contractPath);
+        let Network_type = 'TESTNET';
+
+        if(Helper.isMainnetNode(tezosNode)) {
+            Network_type = 'MAINNET';
+        }
+
+        let conseilServer = { 'url': `${CONSEIL_SERVER[Network_type].url}`, 'apiKey': `${CONSEIL_SERVER[Network_type].apiKey}`, 'network': `${NODE_TYPE[Network_type]}` };
+
+        let contractCode, contractAddress, contractPath;
+        let contractObj = Helper.findKeyObj(this.config.contracts, contract);
         if (contractObj) {
             contractAddress = contractObj.pkh;
         }
+
+        if(contract.startsWith('KT')) {
+            contractAddress = contract;
+        } else {
+            contractPath = contract;
+        }
       
         if (!contractAddress && !fs.existsSync(contractPath)) {
-            Logger.error(`Couldn't find the contract, please make sure contract label or address is correct!`);
+            Logger.error(`Couldn't find the contract, please make sure contract-file-path/contract-label/contract-address is correct.`);
```

Comment:
can you make it more descriptive. 
like please check if contract exists at {contractPath} or contract address is correct